src/server.cpp

```diff
@@ -254,19 +255,19 @@ bool GraftServer::initConfigOption(int argc, const char** argv)
     details::init_log(config, vm);
 
     const boost::property_tree::ptree& server_conf = config.get_child("server");
-    m_configOpts.http_address = server_conf.get<std::string>("http-address");
-    m_configOpts.coap_address = server_conf.get<std::string>("coap-address");
-    m_configOpts.timer_poll_interval_ms = server_conf.get<int>("timer-poll-interval-ms");
-    m_configOpts.http_connection_timeout = server_conf.get<double>("http-connection-timeout");
-    m_configOpts.workers_count = server_conf.get<int>("workers-count");
-    m_configOpts.worker_queue_len = server_conf.get<int>("worker-queue-len");
-    m_configOpts.upstream_request_timeout = server_conf.get<double>("upstream-request-timeout");
-    m_configOpts.data_dir = server_conf.get<std::string>("data-dir");
-    m_configOpts.lru_timeout_ms = server_conf.get<int>("lru-timeout-ms");
+    configOpts.http_address = server_conf.get<std::string>("http-address");
+    configOpts.coap_address = server_conf.get<std::string>("coap-address");
+    configOpts.timer_poll_interval_ms = server_conf.get<int>("timer-poll-interval-ms");
+    configOpts.http_connection_timeout = server_conf.get<double>("http-connection-timeout");
+    configOpts.workers_count = server_conf.get<int>("workers-count");
+    configOpts.worker_queue_len = server_conf.get<int>("worker-queue-len");
+    configOpts.upstream_request_timeout = server_conf.get<double>("upstream-request-timeout");
+    configOpts.data_dir = server_conf.get<std::string>("data-dir");
+    configOpts.lru_timeout_ms = server_conf.get<int>("lru-timeout-ms");
 
     const boost::property_tree::ptree& cryptonode_conf = config.get_child("cryptonode");
-    m_configOpts.cryptonode_rpc_address = cryptonode_conf.get<std::string>("rpc-address");
-    //m_configOpts.cryptonode_p2p_address = cryptonode_conf.get<std::string>("p2p-address");
+    configOpts.cryptonode_rpc_address = cryptonode_conf.get<std::string>("rpc-address");
+    //configOpts.cryptonode_p2p_address = cryptonode_conf.get<std::string>("p2p-address");
```

Comment:
Are we still need this?

Reply:
Commented code with p2p-address has been removed